src/UseCases/ValidateAmount/ValidateAmountUseCase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\UseCases\ValidateAmount;
+
+use WMDE\Fundraising\Frontend\Validation\AmountValidator;
+use WMDE\Fundraising\Frontend\ResponseModel\ValidationResponse;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class ValidateAmountUseCase {
```

Comment:
This usecase is somewhat of an empty partition. What do you think about omitting it in this case, by using the AmountValidator directly in the route?